test/ginkgo-ext/junit_reporter.go

```diff
@@ -0,0 +1,197 @@
+package ginkgoext
+
+/*
+
+JUnit XML Reporter for Ginkgo
```

Comment:
Ir was already in there, see a bit below ;-)

Reply:
Given that this is a fork, it would make it a lot easier to review if the initial code was copied into `ginkgo-ext` (i.e., no functional change at all in the code, just the import dependency  in `test_suite_test.go` , and then you patched the changes on top of it). 

Also, are there any copyright concerns / copyright that need to be appended to the top of the file?